src/backend/main/py.main/caliopen_main/message/parsers/twitter.py

```diff
@@ -55,13 +43,68 @@ def __init__(self, raw_data):
         self.is_received = True  # TODO: handle DM sent by user
         #                                     if broker keeps them when fetching
         self.importance_level = 0
+        self.get_bodies()
+
+    def get_bodies(self):
+        self.body_plain = self.dm["message_create"]["message_data"]["text"]
+
+    @property
+    def subject(self):
+        """
+        tweets don't have subject
+        should we return an excerpt ?
+        """
+        return ''
+
+    @property
+    def size(self):
+        """Get json tweet object size in bytes."""
+        return len(self.dm.as_string())
+
+    @property
+    def date(self):
+        return datetime.fromtimestamp(float(self.dm["created_timestamp"][0:10]),
```

Comment:
I think it could be safier to do float(created_timestamp) / 1000 as it's an unix timestamp in millisecond